src/components/pages/RoleSignup/Applications/AppSuccess.js

```diff
@@ -1,28 +1,47 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Col, Row, Typography, Button } from 'antd';
+import { connect } from 'react-redux';
+import { getProfile } from '../../../../state/actions/userProfile/getProfile';
+import axiosWithAuth from '../../../../utils/axiosWithAuth';
 import { Link } from 'react-router-dom';
 const { Title } = Typography;
 
-const AppSuccess = () => {
+const AppSuccess = ({ isAuthenticated, getProfile }) => {
+  const [user, setUser] = useState({});
+  useEffect(() => {
+    axiosWithAuth()
+      .get(`/profile/current_user_profile/`)
+      .then(user => {
+        setUser(user.data);
+        getProfile(user.data.profile_id);
+      });
+  }, [isAuthenticated]);
+
   return (
     <>
       <Row align="center" gutter={[16, 16]} style={{ marginTop: '5vh' }}>
         <Col span={24} align="middle">
-          <Title level={2}>Thank you!</Title>
+          <Title level={2}>Thank you {user.first_name}!</Title>
         </Col>
         <Col span={24} align="middle">
           <Typography>
-            Your application was successfully submitted! Someone at Underdog
-            Devs will contact you shortly in regards to your application
-            approval.
+            Your application has been received, keep an eye on your email, as
+            we'll email you if your application has been approved.
           </Typography>
         </Col>
-        <Link to="/">
+        <Link to="/landing">
           <Button className="button">Home</Button>
```

Comment:
I like this use of the button as a redirect rather than timing the page out to go back to the main page. The only consideration I would make here is stylistic in that it might be more accessible to have the button filled in rather than outlined. But I also know that the Ant styling has been tricky so I wouldn't blame y'all for not wanting to open that can of worms.